sim/battle-actions.ts

```diff
@@ -842,6 +842,7 @@ export class BattleActions {
 			move.hit = hit;
 			if (move.smartTarget && targets.length > 1) {
 				targetsCopy = [targets[hit - 1]];
+				damage = [damage[hit - 1]];
```

Comment:
This was in practice not breaking anything, but still weird to have the trailing zero in the damage array